crates/atuin/Cargo.toml

```diff
@@ -87,8 +87,11 @@ serde_yaml = "0.9.32"
 sysinfo = "0.30.7"
 regex="1.10.4"
 
-[target.'cfg(any(target_os = "windows", target_os = "macos", target_os = "linux"))'.dependencies]
-cli-clipboard = { version = "0.4.0", optional = true }
+[target.'cfg(any(target_os = "windows", target_os = "macos"))'.dependencies]
+arboard = { version = "3.4", optional = true }
+
+[target.'cfg(target_os = "linux")'.dependencies]
+arboard = { version = "3.4", optional = true, features = ["wayland-data-control"] }
```

Comment:
Not sure about this one, but would want to have at least an option to enable the native wayland support.